onnxruntime/test/providers/cpu/tensor/onehot_op_test.cc

```diff
@@ -40,7 +40,7 @@ TEST(OneHotOpTest, DefaultAxis_float_float_float /*indices, output, depth*/) {
 }
 
 TEST(OneHotOpTest, DefaultAxis_int64_int32_float /*indices, output, depth*/) {
-  OpTester test("OneHot", 9);
+  OpTester test("OneHot", 11);
```

Comment:
so we don't want to keep opset 9 tests? I know opset 11 OneHot is a superset of opset 9 though. One possible thing I thought is: a kind of cpu provider implement OPSET 9 using a way different with 11. Your call for this. :)

Reply:
opset 11 is a superset of opset 9. 
and they share the same code path. 
I think updating the test opset set to 11 is enough to cover the kernel imple.